sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -225,6 +225,18 @@ static const struct dmi_system_id sof_rt5682_quirk_table[] = {
 					SOF_RT5682_SSP_AMP(2) |
 					SOF_RT5682_NUM_HDMIDEV(4)),
 	},
+	{
+		.callback = sof_rt5682_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Google_Rex"),
```

Comment:
This will hit every single follower board, this needs to be scoped better

Reply:
> This will hit every single follower board, this needs to be scoped better

Can we add DMI_OEM_STRING until it's needed? Google_Hatch has only one ALC5682 based DB as well:
https://github.com/thesofproject/linux/blob/ae911df742b473788460a1ad4eadd4ed030cba51/sound/soc/intel/boards/sof_rt5682.c#L141

Reply:
I don't think you can use that either as most OEMs don't set it so it defaults to Google. I'll take a look at the DMI ok Brya tomorrow to see what we can use

Reply:
You should probably use `PRODUCT_NAME` as that should be device specific

Reply:
Side note, I am just worried about another UP4 scenario like TGL where we had to hack the oem name, if you think we can avoid then we should be fine with using this as is.

Reply:
Thank you @cujomalainey for the code review, yes I agree theoretically PRODUCT_NAME is more restrictive than PRODUCT_FAMILY and we are using both in the current code:
https://github.com/thesofproject/linux/commit/2a4b91a26403fa3e7b07271700c3ca7103664bba

In this case PRODUCT_FAMILY is Google_Rex, PRODUCT_NAME is Rex, I still prefer PRODUCT_FAMILY :)